modules/ultra/index/main.nf

```diff
@@ -0,0 +1,37 @@
+process ULTRA_INDEX {
+    tag "$gtf"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::ultra_bioinformatics=0.0.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ultra_bioinformatics:0.0.4.1--pyh5e36f6f_0':
+        'quay.io/biocontainers/ultra_bioinformatics:0.0.4.1--pyh5e36f6f_0' }"
+
+    input:
+    path fasta
+    path gtf
+
+    output:
+    tuple path("*.pickle"), path("*.db"), emit: index
```

Comment:
Files are normally emitted into separate channels. Must these two always be present together?

Reply:
To be usable, the pickle files and the db files have to be both together for aligning with uLTRA align. I thought it would be the best way to keep them together.

Reply:
That's fair. I think it's not a good idea to split files that must be used together.